index.css

```diff
@@ -0,0 +1,1130 @@
+}
+
+.copyright {
+  font-weight: bold;
+}
+
+.facebook {
+  width: 20px;
+  height: 20px;
+}
+
+.page,
+.page_out {
+  color: white;
+}
+
+.header_wrapper {
+  display: none;
+}
+
+.see {
+  display: none;
+}
+
+.btn-wrapper {
+  position: fixed;
+  top: 10px;
+  left: 10px;
+}
+
+a.email {
+  display: block;
+  font-size: 0.8em;
+  font-weight: 700;
+  text-underline-position: under;
+  line-height: 1.5rem;
+  color: red;
+}
+
+a.email:hover {
+  color: blue;
+}
+
+#error {
+  color: red;
+  width: 300px;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.course_lecturers {
+  margin-left: 10px;
+  margin-top: 14px;
+}
+
+.checkers {
+  width: 40px;
+  position: relative;
+}
+
+.lecturer_content {
+  margin-left: 100px;
+}
+
+.hidden_features {
+  display: none;
+}
+
+.dynamic {
+  display: flex;
+}
+
+#lecturer-list {
+  display: flex;
+  flex-direction: column;
+  margin-top: 1.5rem;
+  gap: 2rem;
+}
+
+#lecturer-list li {
+  display: flex;
+  gap: 1.5rem;
+}
+
+#lecturer-list li:nth-child(n + 3) {
+  display: none;
+}
+
+.lecturer-details h3 {
+  font-size: 1rem;
+}
+
+.lecturer-image {
+  background-image: url("images/image-bg.png");
+  background-size: contain;
+  background-position: top left;
+  background-repeat: no-repeat;
+  max-width: 100px;
+  flex: 1;
+}
+
+.lecturer-image img {
+  position: relative;
+  width: 70px;
+  height: 70px;
+  top: 0.73rem;
+  left: 0.8rem;
+}
+
+.lecturer-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  flex: 2.5;
+}
+
+.lecturer-title {
+  font-size: 0.82rem;
+  font-weight: 600;
+  line-height: 1.2rem;
+  color: #ec5242;
+}
+
+hr.grey-line {
+  background-color: #d3d3d3;
+  border: #d3d3d3;
+  height: 0.1rem;
+  width: 5%;
+}
+
+.lecturer-info {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  margin-bottom: 10px;
+}
+
+#hr-hero {
+  display: none;
+}
+
+#arrow {
+  display: none;
+}
+
+.about-partner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+a.link,
+a.linkA,
+a.linkB {
+  color: white;
+  text-decoration: none;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+.feat-heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    display: none;
+  }
+
+  .navbar1 {
+    display: flex;
+    justify-content: space-evenly;
+    width: 100%;
+    padding-bottom: 30px;
+  }
+
+  .navbar1 ul {
+    display: inline-block;
+    column-gap: 40px;
+    justify-content: space-between;
+    margin-top: 30px;
+  }
+
+  .navbar1 a {
+    justify-content: space-evenly;
+    text-decoration: none;
+    color: var(--COLOR1);
+    padding: 15px 15px;
+    margin-top: 30px;
+  }
+
+  .top-nav {
+    display: flex;
+    justify-content: space-around;
+    background-color: #272a31;
+    padding-left: 540px;
+    padding-right: 30px;
+  }
```

Comment:
- [ ] As pointed out earlier by the previous reviewer, kindly reduce the space that is between the social media icons so as to have a design close to the given template. 👍 I will implore you to always pay attention to details, this will really help a lot as a software developer. Kindly take this as avenue to give the best. 👏 

## Your Design
![Untitled](https://user-images.githubusercontent.com/103386183/219867528-358e0ee9-421f-46cf-be38-463493c53551.jpg)